recipe/patches/0027-Don-t-checksharedmods.patch

```diff
@@ -0,0 +1,22 @@
+From 1b6604b52651f3088fe07dde50cfc8093124f68a Mon Sep 17 00:00:00 2001
+From: "Uwe L. Korn" <uwe.korn@quantco.com>
+Date: Fri, 1 Sep 2023 23:32:14 +0200
+Subject: [PATCH 20/21] Don't checksharedmods
```

Comment:
Why is this done?

Reply:
Running these checks on installations fails with the following. I wouldn't expect them to work in a cross-compiled setting ever. 

```
Traceback (most recent call last):
  File "/Users/uwe/mambaforge-x64/conda-bld/python-split_1694676471721/work/Tools/build/check_extension_modules.py", line 25, in <module>
    import pathlib
  File "/Users/uwe/mambaforge-x64/conda-bld/python-split_1694676471721/work/Lib/pathlib.py", line 20, in <module>
    from urllib.parse import quote_from_bytes as urlquote_from_bytes
  File "/Users/uwe/mambaforge-x64/conda-bld/python-split_1694676471721/work/Lib/urllib/parse.py", line 36, in <module>
    import math
ImportError: dlopen(/Users/uwe/mambaforge-x64/conda-bld/python-split_1694676471721/work/build-static/build/lib.darwin-aarch64-3.12/math.cpython-312-darwin.so, 0x0002): tried: '/Users/uwe/mambaforge-x64/conda-bld/python-split_1694676471721/work/build-static/build/lib.darwin-aarch64-3.12/math.cpython-312-darwin.so' (mach-o file, but is an incompatible architecture (have 'arm64', need 'x86_64')), '/System/Volumes/Preboot/Cryptexes/OS/Users/uwe/mambaforge-x64/conda-bld/python-split_1694676471721/work/build-static/build/lib.darwin-aarch64-3.12/math.cpython-312-darwin.so' (no such file), '/Users/uwe/mambaforge-x64/conda-bld/python-split_1694676471721/work/build-static/build/lib.darwin-aarch64-3.12/math.cpython-312-darwin.so' (mach-o file, but is an incompatible architecture (have 'arm64', need 'x86_64')), '/Users/uwe/mambaforge-x64/conda-bld/python-split_1694676471721/work/build-static/Modules/math.cpython-312-darwin.so' (mach-o file, but is an incompatible architecture (have 'arm64', need 'x86_64')), '/System/Volumes/Preboot/Cryptexes/OS/Users/uwe/mambaforge-x64/conda-bld/python-split_1694676471721/work/build-static/Modules/math.cpython-312-darwin.so' (no such file), '/Users/uwe/mambaforge-x64/conda-bld/python-split_1694676471721/work/build-static/Modules/math.cpython-312-darwin.so' (mach-o file, but is an incompatible architecture (have 'arm64', need 'x86_64'))
```

Reply:
@isuruf Is that OK? Should I add a more detailed comment or do you want me to look into this further (if so: where/what?)?

Reply:
Yes, but can you apply the patch for when `build_platform != target_platform and CROSSCOMPILING_EMULATOR == ""` so that the patch is applied only when python is cross compiled?